Tests/Marketplace/upload_packs.py

```diff
@@ -961,7 +967,6 @@ def main():
                                   storage_bucket, is_private_content_updated)
 
     # google cloud bigquery client initialized
```

Comment:
```suggestion
```
Comment irrelevant
and IMO better to move the statistic df next to the bq client init